Resources/Textures/Objects/Weapons/Melee/combat_knife.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken/modified from desertrose at https://github.com/DesertRose2/desertrose/pull/378/commits/9e081bd2a0d7614be5a9dabbca34886ce204105d",
+    "copyright": "Taken/modified from desertrose at https://github.com/DesertRose2/desertrose/pull/378/commits/9e081bd2a0d7614be5a9dabbca34886ce204105d, storage sprite by RiceMar1244",
```

Comment:
Everything else on this PR seems fine, but it seems like the inhand sprites here aren't actually in the linked commit.
...in fact, I'm pretty sure even the object sprites aren't in the commit (or in desert rose for that matter,) but I've tried for like 30 minutes to track them down and have been unable to do so.
You should attribute yourself to having made the knife sprites. Something like:
`` "Taken/modified from desertrose at https://github.com/DesertRose2/desertrose/pull/378/commits/9e081bd2a0d7614be5a9dabbca34886ce204105d, storage & inhand sprites by RiceMar1244",``

This also goes for the survival knife.